api/routes/user.js

```diff
@@ -2,17 +2,18 @@ import Router from 'express';
 import usercontroller from '../controllers/user';
 import sessionController from '../controllers/session';
 import Auth from '../middleware/auth';
+import x from '../middleware/signup-middleware';
 
 const userRoutes = Router();
 
 // users create accounts to use the application
-userRoutes.post('/auth/signup', usercontroller.UserController.RegisterUser);
+userRoutes.post('/auth/signup', x.signup, usercontroller.UserController.RegisterUser);
```

Comment:
@Wabuluka , please make your imports descriptive..   what is `X`?

Reply:
ok thank you for the feedback